data/repos.yml

```diff
@@ -386,6 +495,9 @@
   team: "#data-products"
   type: Data science
 
+# Propose archiving - prototype, not touched since April 2020
+- repo_name: govuk-content-publisher-prototype
```

Comment:
@ollietreend

Reply:
Thanks @ChrisBAshton. I've added this to our Trello board: https://trello.com/c/ij70RR02/1199-decommission-and-archive-govuk-content-publisher-prototype